File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int k=nums.length;
        int num1[]=new int[2 * k];
        for(int i=0;i<k;i++){
            num1[i] = nums[i];
            num1[i + k] = nums[i];
        }
        return num1;
    }
}

File: Data Structures & Algorithms/contains-duplicate-ii/submission-1.cpp
class Solution {
public:
    bool containsNearbyDuplicate(vector<int>& nums, int k) {
        map<int,int> cnt;
        for(int i=0;i<nums.size();i++){
        int a=nums[i];
            if(cnt.find(a)!=cnt.end() and abs(i-cnt[a])<=k){
                return true;
            }
            cnt[a]=i;
        }
        return false;
    }
};